services/frontend-service/src/ui/utils/store.tsx

```diff
@@ -441,6 +460,22 @@ export const useLocksConflictingWithActions = (): AllLocks => {
             });
             return actions.length > 0;
         }),
+        teamLocks: locks.teamLocks.filter((teamLock: DisplayLock) => {
+            const actions = allActions.filter((action) => {
+                if (action.action?.$case === 'deploy') {
+                    const app = action.action.deploy.application;
+                    const env = action.action.deploy.environment;
+                    //If getTeam(app) == lock.team and env == lock.env
```

Comment:
remove dead code please

Reply:
I mean this line `//If getTeam(app) == lock.team and env == lock.env`
Please either rephrase it, to be a useful comment, or remove it.